src/App/App.csproj

```diff
@@ -14,12 +14,12 @@
 
   <ItemGroup>
     <PackageReference Include="HockeySDK.Xamarin" Version="5.2.0" />
-    <PackageReference Include="Plugin.Fingerprint" Version="1.4.9" />
-    <PackageReference Include="Refractored.FloatingActionButtonForms" Version="2.1.0" />
-    <PackageReference Include="Xamarin.Essentials" Version="1.3.1" />
+    <PackageReference Include="Plugin.Fingerprint" Version="2.1.1" />
+    <PackageReference Include="Xamarin.Essentials" Version="1.5.1" />
     <PackageReference Include="Xamarin.FFImageLoading.Forms" Version="2.4.11.982" />
-    <PackageReference Include="Xamarin.Forms" Version="4.4.0.991265" />
-    <PackageReference Include="ZXing.Net.Mobile.Forms" Version="2.1.47" />
+    <PackageReference Include="Xamarin.Forms" Version="4.5.0.356" />
+    <PackageReference Include="ZXing.Net.Mobile" Version="2.4.1" />
```

Comment:
I dont think you need to specify `ZXing.Net.Mobile`. It should get pulled in by `ZXing.Net.Mobile.Forms`

Reply:
It should but for some reason isn't.  I found several instances of others who had to do the same thing with 2.4.1.

Reply:
Ok